scripts/bake-dependencies.sh

```
@@ -0,0 +1,186 @@
+    error "Failed to download build dependencies (clone targets) for release build in $BUILD_PATH"
+fi
+
+# Also download debug WebKit variant for debug builds
+print "Downloading debug WebKit variant..."
+if [ -n "$ABI" ]; then
+    cmake -S "$BUN_REPO_PATH" -B "$BUN_REPO_PATH/build/debug" -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCI=ON -DABI="$ABI"
+else
+    cmake -S "$BUN_REPO_PATH" -B "$BUN_REPO_PATH/build/debug" -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCI=ON
+fi
+
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)